lib/checking_account.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'account'
+
+module Bank
+  class CheckingAccount < Account
+    def initialize (id, balance)
+      super(id, balance)
+      @checks = 0
+    end
+
+    def withdraw(withdraw_amount)
```

Comment:
It might be cleaner to use `super` here to call `Account#withdraw`, and centralize your logic all in one place. In this case `Account#withdraw` would probably require more parameters, like a fee and a minimum balance.